website/src/components/Layout.tsx

```diff
@@ -52,6 +54,12 @@ export const getDashboardLayout = (page: React.ReactElement) => (
           desc: "User Leaderboard",
           icon: FiBarChart2,
         },
+        {
+          label: "Guide",
+          pathname: "/guide",
```

Comment:
There isn't anything at `/guide` is it?

Reply:
There is nothing in /guide, it is under development